docs/server/source/http-client-server-api.rst

```diff
@@ -147,7 +147,17 @@ Transactions
    .. literalinclude:: http-samples/post-tx-response.http
       :language: http
 
+   .. note::
+       Is the server returning a ``202`` HTTP status code, then the transaction
+       has been accepted for processing. To check the status of the
+       transaction, poll the link to the `status monitor
+       <https://docs.bigchaindb.com/projects/server/en/latest/http-client-server-api.html#get--api-v1-statuses?tx_id=tx_id>`_
+       provided in the ``Location`` header or listen to server's `WebSocket
+       Event Stream API
+       <https://docs.bigchaindb.com/projects/server/en/latest/websocket-event-stream-api.html>`_.
```

Comment:
Let's try to avoid hard-coded URLs like this, when possible. Instead you can use something like:
```text
or listen to the server's :ref:`WebSocket Event Stream API <The WebSocket Event Stream API>`.
```